src/waveform/renderers/waveformmarkproperties.h

```diff
@@ -0,0 +1,27 @@
+#ifndef WAVEFORMMARKPROPERTIES_H
+#define WAVEFORMMARKPROPERTIES_H
+
+#include <QColor>
+#include <QDomNode>
+
+class SkinContext;
+class WaveformSignalColors;
+
+class WaveformMarkProperties {
+  public:
+    WaveformMarkProperties();
+    WaveformMarkProperties(const QDomNode& node,
+                           const SkinContext& context,
+                           const WaveformSignalColors& signalColors);
+    virtual ~WaveformMarkProperties();
+
+    QColor m_color;
+    QColor m_textColor;
+    QString m_text;
+    Qt::Alignment m_align;
+    QString m_pixmapPath;
+
+    int m_iHotCue;
```

Comment:
I think this member belongs to WaveformMark instead of WaveformMarkProperties, because it is not an intrinsic property and can only be interpreted with the context at hand.